discretize/__init__.py

```diff
@@ -25,9 +25,13 @@
 
             to compile the cython code.
             """
-            )
+        )
+from discretize import tests
+
+# DERECATED
+Tests = tests
 
-__version__   = '0.5.1'
-__author__    = 'SimPEG Team'
-__license__   = 'MIT'
-__copyright__ = '2013 - 2019, SimPEG Developers, http://simpeg.xyz'
+__version__ = "0.5.1"
+__author__ = "SimPEG Team"
+__license__ = "MIT"
+__copyright__ = "2013 - 2019, SimPEG Developers, http://simpeg.xyz"
```

Comment:
Adjust (automate) year.

Reply:
(I guess generally everything in #111, which includes the removing of `__future__` etc and the automation of the year)